index.html

```diff
@@ -42,12 +42,12 @@ <h3 clothes-title><span class="arrows">➫</span>Accessories</h3>
         <article class="background-selection" id="background-selection">
           <div class="background-header-btns">
             <h3 class= clothes-title><span class="arrows">➪</span>Backgrounds</h3>
-            <button class="favoriteStyled background-Options blue" type="button">Blue</button>
-            <button class="favoriteStyled background-Options park margin-left-class-btn" type="button">Park</button>
-            <button class="favoriteStyled background-Options beach margin-left-class-btn" type="button">Beach</button>
-            <button class="favoriteStyled background-Options space margin-left-class-btn" type="button">Space</button>
-            <button class="favoriteStyled background-Options yellow" type="button">Yellow</button>
-            <button class="favoriteStyled background-Options hearts margin-left-class-btn" type="button">Hearts</button>
+            <button class="favoriteStyled background-Options blue blue-background" type="button">Blue</button>
+            <button class="favoriteStyled background-Options park park-background margin-left-class-btn" type="button">Park</button>
+            <button class="favoriteStyled background-Options beach beach-background margin-left-class-btn" type="button">Beach</button>
+            <button class="favoriteStyled background-Options space space-background margin-left-class-btn" type="button">Space</button>
+            <button class="favoriteStyled background-Options yellow yellow-background" type="button">Yellow</button>
+            <button class="favoriteStyled background-Options hearts heart-background margin-left-class-btn" type="button">Hearts</button>
```

Comment:
Added more specific class name.  Can be changed in future if necessary.

Reply:
Like it. Let's discuss today what we want to do with the converged classes we meshed.

Reply:
Yep, agreed. Lets make sure that we have clear & definitive classes to each element / One class that the CSS interacts with, and One class that the JavaScript interacts with (as needed).